### DISCUSSION TITLE
Lesson 6 - test_only_owner_can_withdraw: How does your error message look like, and if no error, what's your setup?

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1084] [Q&A] [answered]
Edit: Repository with source code can be found here: https://github.com/n4n0b1t3/brownie.reverts.example

In the process of getting to the bottom of the error when using `pytest.raise()` I noticed, that Patrick is getting a different error with an untreated reverted transaction than my setup produces.
This is a screenshot of the tutorial video at the timestamp [5:53:40](https://youtu.be/M576WGiDBdQ?t=21220)
![image](https://user-images.githubusercontent.com/32838961/154480632-42e4cc7e-3b34-46ea-a072-971fb941a441.png)

I am getting a very different result:
![image](https://user-images.githubusercontent.com/32838961/154482441-9430c09f-0118-46a5-9337-59b56fadc5d2.png)

I believe this is caused by `brownie\network\transaction.py` exiting with its own `raise exc._with_attr` instead of "bubbling up" the error to `pytest.raise()`. (Does this hypothesis make any sense?)

Can you try this in your environment? Are you getting the same result or does your message look like the one from the video?

More info and the complete setup and test code can be found here:
https://ethereum.stackexchange.com/questions/121687/
https://stackoverflow.com/questions/71126128/

**Comment by cromewar** [comment]
This is huge, I'll try to reproduce this on my on environment and compare the results!!

**Reply by n4n0b1t3** [reply]
I added the repository https://github.com/n4n0b1t3/brownie.reverts.example

**Reply by n4n0b1t3** [reply]
I just opened an issue on https://github.com/eth-brownie/brownie/issues/1441 and sincerely hope that didn't do any super stupid mistake :-)

**Comment by n4n0b1t3** [comment, marked as answer]
A quite lengthy solution see here: https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/1110
